verify.go

```diff
@@ -65,12 +65,24 @@ func (v *verifier) existsSecretValue(ctx context.Context, from string) error {
 			return errors.New("invalid arn format")
 		}
 		secretArn := strings.Join(part[0:7], ":")
-		_, err := v.secretsmanager.GetSecretValue(ctx, &secretsmanager.GetSecretValueInput{
+		res, err := v.secretsmanager.GetSecretValue(ctx, &secretsmanager.GetSecretValueInput{
 			SecretId: &secretArn,
 		})
 		if err != nil {
 			return fmt.Errorf("failed to get secret value from %s secret id %s: %w", from, secretArn, err)
 		}
+		key := part[7]
```

Comment:
`part` slice contains only 7 elements if json key not specified. So this lookup raise a panic (`runtime error: index out of range [7] with length 7`).

```suggestion
		if len(part) < 8 {
			return nil
		}
		key := part[7]
```